pkg/hostap/monitor.go

```diff
@@ -140,17 +157,30 @@ func (m *InterfaceMonitor) handleHostapdEvent(eventStr string) error {
 	eventStrSlice := strings.Split(eventStr, " ")
 	if len(eventStrSlice) < 2 {
 		if isSolicitedEvent(eventStr) {
+			if strings.Contains(eventStr, statusReply) {
+				m.ifEAPState = getState(eventStr)
```

Comment:
This feels like a bit of an API bleed; we're exposing part of the raw event from hostapd to the end-user; Would it be safer to come up with our own state enum instead?

Reply:
I was just using event string (which contains 'state) as it is to denotate state in k8s. anyway i changed it to use enum now.